### ISSUE TITLE
No error is displayed while pushing the form with blank URL

### ISSUE THREAD

**Issue by shobhitagarwal1612** [opendatakit/briefcase#362]
#### Software versions 
Briefcase v1.9.0, Ubuntu 17.10

#### Problem description
No error is displayed in the UI while pushing the form without setting the URL. Although, in the console I can see the following error : 

```
[pool-1-thread-1] ERROR org.opendatakit.briefcase.util.TransferAction - upload transfer action failed
java.lang.NullPointerException
	at org.opendatakit.briefcase.util.AggregateUtils.testServerConnectionWithHeadRequest(AggregateUtils.java:409)
	at org.opendatakit.briefcase.util.ServerUploader.uploadForm(ServerUploader.java:278)
	at org.opendatakit.briefcase.util.ServerUploader.uploadFormAndSubmissionFiles(ServerUploader.java:206)
	at org.opendatakit.briefcase.util.TransferToServer.doAction(TransferToServer.java:41)
	at org.opendatakit.briefcase.util.TransferAction$UploadTransferRunnable.run(TransferAction.java:58)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

```

#### Steps to reproduce the problem
1. Goto `Push` tab
2. Leave the `URL` blank
3. Select one/more form(s)
4. Click on `Push` button

#### Expected behavior
Push status should display appropriate error message

#### Other information 
If I click on `Connect` button and press `Cancel` in the dialog, then clicking on `Push` button shows an error message

**Comment by mayank8318** [comment]
@shobhitagarwal1612 In PR #337 I am disabling the Push button if no URL is selected. Kindly check that :)

**Comment by icemc** [comment]
@opendatakit-bot claim

**Comment by icemc** [comment]
Hello @mayank8318 are you working on this issue?

**Comment by icemc** [comment]
Just noticed your comment above

**Comment by mayank8318** [comment]
@icemc Yes in my PR this issue will not happen.

**Comment by ggalmazor** [comment]
I think this issue was solved by #337. @shobhitagarwal1612, can you confirm it and close it, please?

**Comment by yanokwa** [comment]
Agreed with @ggalmazor that this was solved.
